plugins/callback/logstash.py

```diff
@@ -19,16 +19,26 @@
         description: Address of the Logstash server
         env:
           - name: LOGSTASH_SERVER
+        ini:
+          - section: callback_logstash
+            key: server
+            version_added: 1.0.0
         default: localhost
       port:
         description: Port on which logstash is listening
         env:
             - name: LOGSTASH_PORT
+        ini:
+          - section: callback_logstash
+            key: port
```

Comment:
You also need a `version_added` here, and also for the third ini entry.

Reply:
ok